src/pages/Committees.js

```diff
@@ -554,14 +614,14 @@ function IOR() {
             The Information Organ (Swedish: Informationsorganet, IOR) is a very important committee inside the chapter,
             as it is responsible for developing and maintaining the chapter's systems, including
             the <a href="https://github.com/datasektionen/">GitHub</a> that hosts this website code. One of the most
-            important systems is <a href="https://cashflow.datasektionen.se/">CachFlow</a> where you can ask to be refunded
+            important systems is <a href="https://cashflow.datasektionen.se/">CashFlow</a> where you can ask to be refunded
```

Comment:
```suggestion
            important systems is <a href="https://cashflow.datasektionen.se/">CashFlow</a> where you can ask to be reimbursed
```

Reply:
Are you sure "reimbursed" is the most common way to say it? It feels too French for me.

Reply:
Given that I'm not French, yes I'm quite sure. A refund is usually when you're not satisfied with what you bought so you take it back to the shop. A reimbursement is compensation for unduly lost money, such as for an approved expense.